### DISCUSSION TITLE
FSRS is coming soon. So how soon can it be done?

### DISCUSSION THREAD

**Discussion by ququ** [st3v3nmw/obsidian-spaced-repetition#1108] [Q&A]
First of all, I would like to express my sincere gratitude for providing such a plugin for us to use. I am currently preparing for an exam. I really need the FSRS algorithm to alleviate my review burden. In your readme file, you mentioned that the FSRS algorithm is coming soon. Could you please specify when exactly the algorithm will be implemented?

**Comment by KyleKlus** [comment]
For all updates on this feature please visit this #1397 issue 😊

In general I would definitely say not before the end of this year
